frontend-react/src/hooks/UseOktaMemberships.test.ts

```diff
@@ -212,6 +235,16 @@ describe("useOktaMemberships", () => {
 });
 
 describe("helper functions", () => {
+    describe("getTypeOfGroup", () => {
+        test("returns the right type of group", () => {
+            const admin = getTypeOfGroup("DHPrimeAdmins");
```

Comment:
PEDANTIC POINT, but we could get rid of the variables and just asset against direct invocations:

```js
expect(getTypeOfGroup("DHPrimeAdmins")).toEqual("prime-admin")
expect(getTypeOfGroup("DHSender_test-group")).toEqual("sender")
expect(getTypeOfGroup("DHtest-group")).toEqual("receiver")
```

Also, we may want to add another test case or two for the fall-through:
```js
expect(getTypeOfGroup("")).toEqual("non-standard")
expect(getTypeOfGroup("whatever")).toEqual("non-standard")
```
This case may not actually be encountered in production, but it's still a possibility in our code so I feel that warrants a test 🤷

Reply:
Oh crap, good catch! We should definitely be testing "non-standard" as it's the catch-all for misconfigured groups, or groups like `simple_report`, purely existing for auth, but do not need functionality given to them in our web application